### PR TITLE
Higher learning rate (0.006 → 0.008) with adjusted warmup

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The model is still improving at epoch 70, suggesting it hasn't fully converged. A higher LR (0.008 vs 0.006) with gradient clipping should accelerate convergence, effectively squeezing more learning into the same epoch budget. With only 1 layer and gradient clipping at max_norm=1.0, the model is unlikely to diverge — clipping provides a safety net.

The warmup start_factor also needs to scale proportionally to maintain the warmup ratio.

## Instructions

In `train.py`, make these changes:

### 1. Learning rate (line 28):
```python
    lr: float = 0.008            # was 0.006
```

### 2. Warmup start factor (line 83):
```python
warmup = LinearLR(optimizer, start_factor=1e-5/0.008, total_iters=5)
```

That's it — two small changes. Everything else stays the same.

W&B tag: `mar14b`, group: `lr-008`

## Baseline
- **surf_p = 40.1**, surf_Ux = 0.57, surf_Uy = 0.31
- lr=0.006, 1-layer single-head model, 70 epochs at 4s/epoch

---

## Results
_To be filled by student_

### CODE REVIEW

**Comment by tcapelle** [comment]
## Results

**W&B run:** 2il9afld

| Metric | Baseline (lr=0.006) | This run (lr=0.008) |
|--------|---------------------|---------------------|
| surf_p | 40.1 | **39.4** |
| surf_Ux | 0.57 | 0.58 |
| surf_Uy | 0.31 | 0.31 |
| val/loss | 0.728 | 0.758 |

**Speed:** 4s/epoch, 70 epochs in 5 min (identical to baseline)

**Outcome: Marginally Positive** — surf_p improved slightly (40.1 → 39.4, ~1.7%). Val/loss is slightly worse due to the higher LR causing marginally more overfitting. Training stable with gradient clipping. Still converging at epoch 70 (epoch 70 shows surf_p=39.1), suggesting further improvement with more epochs.
